scripts/migrations/024_update_type_from_obligations_to_obligation.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2020. Part of the SW360 Portal Project.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# This is a manual database migration script. It is assumed that a
+# dedicated framework for automatic migration will be written in the
+# future. When that happens, this script should be refactored to conform
+# to the framework's prerequisites to be run by the framework. For
+# example, server address and db name should be parameterized, the code
+# reorganized into a single class or function, etc.
+#
+# This script updates type of document from obligations to obligation
+# ---------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+import time
+import couchdb
+import json
+from webbrowser import get
+
+# ---------------------------------------
+# constants
+# ---------------------------------------
+
+DRY_RUN = True
+
+COUCHSERVER = "http://localhost:5984/"
+DBNAME = 'sw360db'
+
+couch=couchdb.Server(COUCHSERVER)
+db = couch[DBNAME]
+
+# set fieldName
+newValue = "obligation"
+
+# ----------------------------------------
+# queries
+# ----------------------------------------
+
+# get all obligations
+all_obligations_query = {"selector": {"type": {"$eq": "obligations"}}}
+
+# ---------------------------------------
+# functions
+# ---------------------------------------
+
+def run():
+    log = {}
+    count = 0;
+    log['updatedObligation'] = []
+    print 'Getting all obligations'
+    all_obligations = db.find(all_obligations_query)
+    print 'found ' + str(len(all_obligations)) + ' obligations in db!'
+    log['totalCount'] = len(all_obligations)
+
+    for obligation in all_obligations:
+        print '\tUpdating type of document from obligations to '+newValue+' for ID -> ' + obligation.get('_id')
+        obligation['type'] = newValue
+        updatedObligation = {}
+        updatedObligation['id'] = obligation.get('_id')
+        log['updatedObligation'].append(updatedObligation)
+        count = count + 1
```

Comment:
Please remove this count, It seems like it is not used anywhere.

Reply:
removed count variable